### PR TITLE
BD-1028: Compliance Issues in Lightning gh-pages repo

### PR DESCRIPTION
**Pull request by mallikarjuna54**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -199,3 +199,10 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
```
Comment (mhughesacn) [inline LICENSE:202]: Please revert this change and append a copy of the MIT license.

**Review by mhughesacn — COMMENTED** [review]
Hi Mallikarjuna - a few comments here, but please see the BD ticket.

LICENSE as below.

README - please revert this change.

lightning-inspect.js - please revert; this looks like a typo.

File: devtools/lightning-inspect.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+window.attachInspector = function({Element, ElementCore, Stage, Component, ElementTexturizer, Texture}) {
```
Comment (mhughesacn) [inline devtools/lightning-inspect.js:1]: This must be reverted - not sure where it came from.

**Review by mhughesacn — CHANGES_REQUESTED** [review]

**Comment by pradeeptakdas** [comment]
Hi @wouterlucas 
Could you please review and merge this PR?

**Comment by pradeeptakdas** [comment]
Hi @wouterlucas , 
Can you please merge this change-set? or Please suggest if you have any review comments.

**Pull request merged**